src/jit/emit.h

```diff
@@ -983,8 +983,32 @@ class   emitter
         void            idCodeSize(unsigned sz) { _idCodeSize = sz; assert(sz == _idCodeSize);     }
 
 #elif defined(_TARGET_ARM64_)
-
-        unsigned        idCodeSize() const      { return 4; }
+        unsigned        idCodeSize() const      {
+                                                    int size = 4;
+                                                    switch (idInsFmt())
+                                                    {
+                                                        case IF_LARGEADR:
+                                                            // adrp + add
+                                                        case IF_LARGEJMP:
+                                                            // b<cond> + b<uncond>
+                                                            size = 8;
+                                                            break;
+                                                        case IF_LARGELDC:
+                                                            if (isVectorRegister(idReg1()))
+                                                            {
+                                                                // adrp + ldr + fmov
```

Comment:
The comment is meant for `ldr` as it is instead of `add`. The change is from the previous commit. As mentioned in this PR, I'd like the latter part (https://github.com/dotnet/coreclr/pull/4919/commits/3173d5b58dd410703ac67d29ba68c6dbb87879e4) to be reviewed in this PR. Can you please review the previous CR first - 
https://github.com/dotnet/coreclr/pull/4896 ?